tests/utils/test_arguments_converter.py

```diff
@@ -2,17 +2,17 @@
 from typing import List, Optional
 
 import strawberry
-from strawberry.arguments import UNSET, ArgumentDefinition, convert_arguments
+from strawberry.arguments import UNSET, StrawberryArgument, convert_arguments
 
 
 def test_simple_types():
     args = {"integer": 1, "string": "abc", "float": 1.2, "bool": True}
 
     arguments = [
-        ArgumentDefinition(name="integer", type=int, origin_name="integer"),
```

Comment:
Is it necessary for these tests to specify _both_ `python_name` and `graphql_name`?

Reply:
updated to pass graphql_name=None :)